website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
Also using index view in forgot_password.mako template
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/public/forgot_password.mako#L27